.travis/deploy-storybook.sh

```diff
@@ -13,6 +13,7 @@ echo "Bucket directory: ${BUCKET_DIR_NAME}"
 npm run storybook-build
 
 # Upload the files
+gsutil -m rsync -r .storybook-out/favicon.ico "gs://${GS_BUCKET_NAME}/favicon.ico"
```

Comment:
Use `gsutil cp .storybook-out/favicon.ico "gs://${GS_BUCKET_NAME}/favicon.ico"` here, since you don't need rsync, recursion, or multithreaded.

Reply:
7327fb8 :)